crates/voicevox_core/src/publish.rs

```diff
@@ -341,6 +340,14 @@ impl InferenceCore {
         self.status_option = None;
     }
 
+    pub fn metas(&mut self) -> &CString {
+        if let Some(status) = self.status_option.as_mut() {
+            &status.metas_str
+        } else {
+            &EMPTY_METAS_CSTRING
+        }
+    }
```

Comment:
`&CStr`を返す方が慣習的なんじゃないかと思います。

```suggestion
    pub fn metas(&mut self) -> &CStr {
        if let Some(status) = self.status_option.as_mut() {
            &status.metas_str
        } else {
            CStr::from_bytes_with_nul(b"\0").unwrap()
        }
    }
```

(このsuggestionを適用する場合、↑の`get_metas_json`の返り値の変更と↓の`EMPTY_METAS_CSTRING`の削除も必要です)

Reply:
ありがとうございます...!
適用してみます！